cmd/srcd/cmd/root.go

```
@@ -80,3 +85,35 @@ func initConfig() {
 		fmt.Println("Using config file:", viper.ConfigFileUsed())
 	}
 }
+
+var logMsgRegex, _ = regexp.Compile(`.*msg="(.+)"`)
```

Use `regexp.MustCompile(` instead of ignoring the error silently